solidity/ts/testsuite/simulator/utils/constants.ts

```diff
@@ -21,6 +23,12 @@ export const ENS_ETHEREUM_NAME_SERVICE = 0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147e
 export const ENS_PUBLIC_RESOLVER_2 = 0x4976fb03C32e5B8cfe2b6cCB31c09Ba78EBaBa41n
 export const ENS_REGISTRY_WITH_FALLBACK = 0x00000000000C2E074eC69A0dFb2997BA6C7d2e1en
 export const ENS_REVERSE_REGISTRAR = 0xa58E81fe9b61B5c3fE2AFD33CF304c454AbFc7Cbn
+export const PROXY_DEPLOYER_ADDRESS = 0x7a0d94f55792c434d74a40883c6ed8545e406d12n
+export const VITALIK = 0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045n
+export const RICH_ADDRESS_1 = 0xD1668fB5F690C59Ab4B0CAbAd0f8C1617895052Bn
+export const RICH_ADDRESS_2 = 0x28Bfd713962f86f2Ab23C59B3b6e1193C54aA097n
+export const RICH_ADDRESS_3 = 0x649614d3f7A7d68b3Cf3C00264a07eeae4326fdcn;
+export const RICH_ADDRESS_4 = 0x14468ADbc206898D9ea1F94009F24Bb335B1e4E7n;
```

Comment:
Rather than finding and hard coding some wealthy addresses, we should use `eth_simulate` state overrides to just make some addresses of our choosing rich during simulations.  @KillariDev knows how to do this as we do it in Interceptor.

Reply:
here's code from Tronar to mint funds to address:
```ts
const mintTorn = async (mockWindowEthereum: MockWindowEthereum, mintAmounts: { address: EthereumAddress, amount: EthereumQuantity }[]) => {
	const tornAddress = CONTRACTS.mainnet.governance['TORN Token']
	const overrides = mintAmounts.map((mintAmounts) => {
		const encodedKeySlotHash = keccak256(encodeAbiParameters([{ type: 'address' }, { type: 'uint256' }], [addressString(mintAmounts.address), 0n]))
		return { key: encodedKeySlotHash, value: mintAmounts.amount }
	})
	const stateSets = overrides.reduce((acc, current) => {
		acc[current.key] = current.value
		return acc
	}, {} as { [key: string]: bigint } )
	await mockWindowEthereum.addStateOverrides({ [addressString(tornAddress)]: { stateDiff: stateSets }})
}
```